changelogs/fragments/3231-fix-keycloak-realm-events.yml

```diff
@@ -0,0 +1,5 @@
+bugfixes:
+  - keycloak_realm - Element type for ``events_listeners`` parameter should be ``string`` instead of ``dict``.
```

Comment:
```suggestion
  - keycloak_realm - element type for ``events_listeners`` parameter should be ``string`` instead of ``dict`` (https://github.com/ansible-collections/community.general/pull/3231).
```